cluster/manifests/wiz/sensor/dameonset.yaml

```diff
@@ -0,0 +1,220 @@
+            - DAC_READ_SEARCH # file hashing
+            - NET_ADMIN # network events
+            - NET_RAW # network events
+            - IPC_LOCK # eBPF
+            - FOWNER # file hashing
+            - SYS_PTRACE # eBPF
+            - SYSLOG # kernel symbol resolve
+            - KILL # response
+          privileged: false
+          runAsNonRoot: true
+          runAsUser: 2202
+          runAsGroup: 2202
+          readOnlyRootFilesystem: true
+          seccompProfile:
+            type: Unconfined
+          seLinuxOptions:
+            user: "system_u"
+            role: "system_r"
+            level: "s0"
+            type: "spc_t"
+
+        env:
+        - name: MIN_SENSOR_COMPAT_BUILD_VER
+          value: "4142"
+        - name: HELM_CHART_COMPAT_VER
+          value: "1"
+        - name: WIZ_TMP_STORE
+          value: "/wiz-sensor-store/"
+        - name: WIZ_HOST_STORE
+          value: "/wiz-host-cache/"
+        - name: WIZ_RAMFS_STORE
+          value: "/tmp/"
+        - name: LOG_FILE
+          value: "/wiz-sensor-store/sensor.log"
+        - name: MY_NODE_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: spec.nodeName
+        - name: MY_POD_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.name
+        - name: MY_POD_UID
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.uid
+        - name: MY_POD_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        - name: MY_APIKEY_SECRET_NAME
+          value: wiz-sensor-apikey
+        - name: AWS_EC2_METADATA_DISABLED
+          value: "true"
+        - name: CRI_SOCKET_CUSTOM_PATH
+          value: 
+        - name: HTTP_PROXY_URL
+          value: 
+        - name: HTTP_PROXY_USERNAME
+          value: 
+        - name: HTTP_PROXY_PASSWORD
+          value: 
+        - name: HTTP_PROXY_CERT
+          value: 
+        - name: BACKEND_ENV
+          value: prod
+        - name: SENSOR_CLUSTER_NAME
+          value: 
+        - name: RUST_LOG
+          value: info
+        - name: STDOUT_LOG
+          value: error
+        - name: POD_MEM_LIMITS
+          valueFrom:
+            resourceFieldRef:
+              containerName: wiz-sensor
+              resource: limits.memory
+              divisor: "1Mi"
+        - name: POD_CPU_LIMITS
+          valueFrom:
+            resourceFieldRef:
+              containerName: wiz-sensor
+              resource: limits.cpu
+              divisor: "1m"
+
+        volumeMounts:
+        - name: sensor-host-cache
+          mountPath: /wiz-host-cache/
+        - name: sensor-store
+          mountPath: /wiz-sensor-store/
+        - name: tmp-store
+          mountPath: /tmp/
+        - name: api-client-secret
+          mountPath: /api-client/
+        - name: api-endpoint-name-secret
+          mountPath: /api-endpoint-name/
+        resources:
+          limits:
+            cpu: 300m
```

Comment:
Lets set this to be the same as the `requests` to be consistent, even though the limits are disabled.